plugins/coverimage.koplugin/main.lua

```diff
@@ -193,6 +227,70 @@ function CoverImage:addToMainMenu(menu_items)
                     end
                 end,
             },
+            -- menu entry: scale book cover
+            {
+                text = _("Scale book cover"),
+                help_text = _("If disabled, the cover will have to be scaled by the FW.\nDisabling is the fastest method, but doesn't work on all devices."),
+                checked_func = function()
+                    return self.ui and self.ui.doc_settings and self.cover_image_background ~= "none"
```

Comment:
No need to check for `self.ui` and `self.ui.doc_settings` here now :)

Reply:
Right!